doc/source/whatsnew/v3.0.0.rst

```diff
@@ -30,6 +30,7 @@ Other enhancements
 ^^^^^^^^^^^^^^^^^^
 - :class:`pandas.api.typing.FrozenList` is available for typing the outputs of :attr:`MultiIndex.names`, :attr:`MultiIndex.codes` and :attr:`MultiIndex.levels` (:issue:`58237`)
 - :class:`pandas.api.typing.SASReader` is available for typing the output of :func:`read_sas` (:issue:`55689`)
+- :meth:`pandas.api.interchange.from_dataframe` now uses the [Arrow PyCapsule Interface](https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html) if available, only falling back to the Dataframe Interchange Protocol if that fails (:issue:`60739`)
```

Comment:
You'll probably need to use the rst version of making `Arrow PyCapsule Interface` hyperlink

Reply:
thanks, been too long since i made a PR 🤦 😳